collectors/ciscoduo/collector.js

```diff
@@ -98,8 +98,9 @@ class CiscoduoCollector extends PawsCollector {
         tsPaths = objectDetails.tsPaths;
 
         const stateMaxtime = state.maxtime ? `till ${moment(parseInt(state.maxtime)).toISOString()}` : ``;
+        const stateMintime = state.stream === Authentication ? moment(parseInt(state.mintime)).toISOString() : moment(parseInt(state.mintime * 1000)).toISOString(); // Convert Epoch timestamp to milliseconds to get date in correct format
```

Comment:
should we do the same for `stateMaxtime ` above?

Reply:
maxtime we get only for Authentication stream. The value is already in milliseconds.

Reply:
should we have same conversion for `state.maxtime` above?

Reply:
Maxtime we get only for Authentication stream which we format at line100 and for rest streams[Administrator, Telephony, OfflineEnrollment] we set maxtime empty `''` to display.